csi/node_server.go

```diff
@@ -280,6 +282,16 @@ func (ns *NodeServer) nodeStageSharedVolume(volumeID, shareEndpoint, targetPath
 			if err == nil {
 				return nil
 			}
+			// Still failed.  Log with mounting node and kernel version for possible troubleshooting.  Don't step on actual mount error.
+			kernelRelease, err1 := lhns.GetKernelRelease()
+			if err1 != nil {
+				kernelRelease = err1.Error()
+			}
+			osDistro, err2 := lhns.GetOSDistro()
+			if err2 != nil {
+				osDistro = err2.Error()
+			}
+			log.WithError(err).Warnf("Failed to mount volume %v on node %s with kernel release %s, os distro %s.", volumeID, ns.nodeID, kernelRelease, osDistro)
```

Comment:
NIT: I guess it isn't called out in https://github.com/longhorn/longhorn/wiki/coding-convention#wording, but our log messages virtually never end with punctuation.

Reply:
I ended up just changing https://github.com/longhorn/longhorn/wiki/Coding-Convention#wording to reflect my perception of our convention. Feel free to change it back if you disagree.

Reply:
Ok, thanks for the heads-up.  I pushed it, and that cleared your approval, but thanks for taking a look.